draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,46 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be a router that captures packets on all its
+            interfaces in both directions. When a packet hits interface A
+            on ingress, an EPB entry gets created, TTL gets decremented,
+            and right before it egresses on interface B another EPB entry
+            gets created in the trace file. In this case, two packets are
+            in the capture file, which are not identical but the
+            epb_packetid can be used to correlate them.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_queue:"><vspace blankLines="0"/>The epb_queue
+            option is a 32-bit unsigned integer that identifies on which queue
+            of the interface the specific packet was received.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_verdict:"><vspace blankLines="0"/>The epb_verdict
+            option stores a verdict of the packet. The verdict indicates what
+            would be done with the packet after processing it. For example, a
+            firewall implemented with eBPF could drop the packet. This verdict
```

Comment:
I'd just say "a firewall could drop the packet"; how the firewall rules are internally implemented is a detail.

Reply:
Will remove it!